src/Gtk/Avalonia.Gtk3/Interop/Native.cs

```diff
@@ -20,6 +20,24 @@ static class Native
     {
         public static class D
         {
+            [UnmanagedFunctionPointer(CallingConvention.Cdecl), GtkImport(GtkDll.Gdk)]
+            public delegate gint16 gdk_display_get_n_screens(IntPtr display);
+
+            [UnmanagedFunctionPointer(CallingConvention.Cdecl), GtkImport(GtkDll.Gdk)]
+            public delegate GtkScreen gdk_display_get_screen(IntPtr display, gint16 num);
```

Comment:
According to https://developer.gnome.org/gdk3/stable/GdkDisplay.html#gdk-display-get-screen
```
gdk_display_get_screen has been deprecated since version 3.20 and should not be used in newly-written code.
There is only one screen; use gdk_display_get_default_screen() to get it.
```

Reply:
Same with `gdk_display_get_n_screens`

Reply:
You also have to make sure that handle returned by `gdk_display_get_default_screen` is unowned, it will be released by finalizer otherwise.